docs/source/getting-started/tutorial.rst

```diff
@@ -51,14 +51,19 @@ and the commands to run:
       --output_dir /tmp/imdb/ --overwrite_output_dir \
       --fp16
 
+.. tip::
+
+  In the YAML, the ``workdir`` and ``file_mounts`` fields are commented out. To
+  learn about how to use them to mount local dirs/files or object store buckets
+  (S3, GCS, R2) into your cluster, see :ref:`sync-code-artifacts`.
```

Comment:
```suggestion
  learn about how to use them to sync local dirs/files or mount your cluster
  with object store buckets(S3, GCS, R2), see :ref:`sync-code-artifacts`.
```
nit: tried to make a clear distinction between mounting and just syncing.

Reply:
I think this should be fine -- the vanilla `file_mounts` uses the word "mount" anyway. Would be good to keep concepts simple here since the purpose it to add a pointer.